.build.config.js

```diff
@@ -1,31 +0,0 @@
-var srcFolder = "./src";
```

Comment:
Ok, just verified. There is no change to this file, so it shouldn't hold the PR back. But it would be nice to keep history